build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -266,15 +207,14 @@ public static BuildTargetResult PublishSDKSymbolsArchiveToAzure(BuildTargetConte
         public static BuildTargetResult PublishSdkDebToDebianRepo(BuildTargetContext c)
         {
             var version = CliNuGetVersion;
-
             var packageName = Monikers.GetSdkDebianPackageName(c);
             var installerFile = c.BuildContext.Get<string>("SdkInstallerFile");
-            var uploadUrl = AzurePublisherTool.CalculateInstallerUploadUrl(installerFile, Channel, version);
+            string url = AzurePublisherTool.UploadFile(installerFile, AzurePublisher.Products.SDK, version);
```

Comment:
This file is already uploaded above in  AzurePublisherTool.UploadFile(installerFile, AzurePublisher.Products.SDK, version);